localisation/00_wc_wonders_upgrades.csv

```diff
@@ -42,8 +42,8 @@ upgrade_collection_architecture_desc;The secrets of building great edifices, roa
 upgrade_collection_math_desc;From the works of Kaldorei thinkers, to the most recent treatises on mathematics.;;Von den Werken der kaldoreiischen Denker bis hin zu den neuesten Abhandlungen �ber Mathematik.;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
 #Historical wonder upgrades;;;;;;;;;;;;;;x
-upgrade_forge_of_wills;Forge of Wills;;Seelenschmiede;;;;;;;;;;;x
-upgrade_forge_of_wills_desc;The Forge of Wills accessed in the Halls of Stone alongside the Forge of Origination in Uldum are great machines devised to empower Azeroth.  Stretching underneath the Storm Peaks the titanic watchers Mimiron and Archaedas built the Forge of Wills. The forge infuses Azeroth's world-soul with cosmic energies to shape her sentience and through the use of Azeroth's life essence sentient life can be created. This is how the titan-forged and their Curse of Flesh afflicted descendants came to be.;;Die Seelenschmiede wurde von den Titanen erbaut, um das schlummernde Azeroth zu st�rken. Sie kann ihre Energien auch dazu nutzen, Legionen von titanischen Kriegern zu formen.;;;;;;;;;;;x
+upgrade_forge_of_wills;Forge of Wills;;Schmiede des Willens;;;;;;;;;;;x
+upgrade_forge_of_wills_desc;The Forge of Wills, accessed from the Halls of Stone,  is one of two great machine, alongside the Forge of Origination in Uldum, devised to empower Azeroth.  Stretching underneath the Storm Peaks, the titanic watchers Mimiron and Archaedas built the Forge of Wills. The forge infuses Azeroth's world-soul with cosmic energies to shape her sentience and through the use of Azeroth's life essence, sentient life can be created. This is how the titan-forged and their descendants, who were afflicted by the Curse of Flesh, came to be.;;Die Schmiede des Willens, die von den Hallen des Steins aus zug�nglich ist, ist eine von zwei gro�en Maschinen neben der Schmiede des Ursprungs in Uldum, die entwickelt wurde, um Azeroth zu st�rken.  Unterhalb der Sturmgipfel errichteten die titanischen W�chter Mimiron und Archaedas die Schmiede des Willens. Die Schmiede f�llt die Weltseele Azeroths mit kosmischen Energien, um ihr Bewusstsein zu formen, und durch die Verwendung von Azeroths Lebensessenz kann f�hlendes Leben geschaffen werden. So entstanden die Titanengeschmiedeten und ihre Nachkommen, die vom Fluch des Fleisches heimgesucht wurden.;;;;;;;;;;;x
```

Comment:
```suggestion
upgrade_forge_of_wills_desc;The Forge of Wills, accessed in the Halls of Stone, alongside the Forge of Origination in Uldum are great machines devised to empower Azeroth.  Stretching underneath the Storm Peaks, the titanic watchers Mimiron and Archaedas built the Forge of Wills. The forge infuses Azeroth's world-soul with cosmic energies to shape her sentience, and through the use of Azeroth's life essence, sentient life is created. This is how the titan-forged and their Curse of Flesh afflicted descendants came to be..;;Die Schmiede des Willens, die von den Hallen des Steins aus zug�nglich ist, ist eine von zwei gro�en Maschinen neben der Schmiede des Ursprungs in Uldum, die entwickelt wurde, um Azeroth zu st�rken.  Unterhalb der Sturmgipfel errichteten die titanischen W�chter Mimiron und Archaedas die Schmiede des Willens. Die Schmiede f�llt die Weltseele Azeroths mit kosmischen Energien, um ihr Bewusstsein zu formen, und durch die Verwendung von Azeroths Lebensessenz kann f�hlendes Leben geschaffen werden. So entstanden die Titanengeschmiedeten und ihre Nachkommen, die vom Fluch des Fleisches heimgesucht wurden.;;;;;;;;;;;x
```
Is the original with grammar and punctuation changes. It gets the idea presented across in fewer words.